subsys/bluetooth/host/conn.c

```diff
@@ -3017,6 +3052,7 @@ int bt_conn_init(void)
 	/* Initialize background scan */
 	if (IS_ENABLED(CONFIG_BT_CENTRAL)) {
 		for (i = 0; i < ARRAY_SIZE(acl_conns); i++) {
+			atomic_set(&acl_conns[i].ref, 0);
```

Comment:
Why is this added? It should not be needed, as the reference count should always be 0 here in the first place.